src/deploy/hosting/release.ts

```diff
@@ -15,38 +14,40 @@ export async function release(context: Context, options: Options, payload: Paylo
     return;
   }
 
-  const projectNumber = await needProjectNumber(options);
-
   logger.debug(JSON.stringify(context.hosting.deploys, null, 2));
   await Promise.all(
     context.hosting.deploys.map(async (deploy) => {
       utils.logLabeledBullet(`hosting[${deploy.site}]`, "finalizing version...");
 
-      const config = await convertConfig(context, payload, deploy.config, true);
-      const data = { status: "FINALIZED", config };
-      const queryParams = { updateMask: "status,config" };
-
-      const finalizeResult = await client.patch(`/${deploy.version!}`, data, { queryParams });
-
-      logger.debug(`[hosting] finalized version for ${deploy.site}:${finalizeResult.body}`);
+      const update: Partial<api.Version> = {
+        status: "FINALIZED",
+        config: await convertConfig(context, payload, deploy.config, /* finalize= */ true),
+      };
+
+      const parts = deploy.version!.split("/");
```

Comment:
Can we just check for `deploy.version`?

Reply:
I assume you mean check instead of using the null assertion. That's fixed in the upstream PR and merged into here.